lib/jasmine.js

```diff
@@ -86,18 +86,21 @@ Jasmine.prototype.addMatchers = function(matchers) {
 
 Jasmine.prototype.loadSpecs = function() {
   this.specFiles.forEach(function(file) {
+    delete require.cache[require.resolve(file)];
```

Comment:
I'm assuming we're deleting the `require.cache` here for a reason, but I'm not sure what it is.

Reply:
If you want to reuse the existing process after finishing one spec, you need to delete the cache. Otherwise, re-require the same file will not work. Actually, only `loadHelper` and `loadRequires` need this. Because of the consistency, I put it here as well

Reply:
This is causing duplicate specs when jasmine.loadSpecs() is called in a reporter. I can see the need for `loadRequires` and `leadHelper` but this should probably be removed from `loadSpec`. I can't see any reason you would want to load the same specs more than once.

Reply:
@wood1986 I'm trying to figure out what to do about this for ES modules, where there's no stable API for manipulating the cache. Can you provide more information about what goes wrong if you re-require the same file without clearing the cache? So far, I haven't been able to come up with any externally-visible difference in behavior.

Reply:
Here is the details. https://github.com/jasmine/jasmine-npm/issues/152#issuecomment-474220861

Reply:
Just for consistency reason. As I said, it can be removed